### PR TITLE
Adarsh_Ovhal_99 & 119

### PR DESCRIPTION
**Pull request by Adarshovhal**

### CODE REVIEW

File: SubmittedAssignments/Adarsh_Ovhal_119/css/style.css [hunk 1]
```diff
@@ -0,0 +1,373 @@
...
+        height: 1mm;
+        margin: 4mm;
+        background-color: rgb(243, 171, 70);
+        position: relative;
+        bottom: 126mm;
+        z-index: 2;
+      }
+      .card1-content
+{
+    margin-left: 23px;
```
Comment (KalpeshApnasite) [inline SubmittedAssignments/Adarsh_Ovhal_119/css/style.css:75]: do not use px unit

File: SubmittedAssignments/Adarsh_Ovhal_119/css/style.css [hunk 2]
```diff
@@ -0,0 +1,373 @@
...
+  font-size: 5pt;
+  font-weight: bold;
+  color: black;
+  }
+
+  .org-phone
+  {
+    position: relative;
+    bottom: 210pt;
+    left: -60px;
```
Comment (KalpeshApnasite) [inline SubmittedAssignments/Adarsh_Ovhal_119/css/style.css:317]: do not use px unit

File: SubmittedAssignments/Adarsh_Ovhal_119/css/style.css [hunk 3]
```diff
@@ -0,0 +1,373 @@
...
+    font-style: normal;
+    font-size: 5.5pt;
+  
+  }
+
+  .org-address
+  {
+    position: relative;
+    bottom: 133pt;
+    left: 73px;
```
Comment (KalpeshApnasite) [inline SubmittedAssignments/Adarsh_Ovhal_119/css/style.css:328]: do not use px unit

**Review by KalpeshApnasite — CHANGES_REQUESTED** [review]

File: SubmittedAssignments/Adarsh_Ovhal_119/css/style.css (hunk 2 above)
Comment (Adarshovhal) [reply SubmittedAssignments/Adarsh_Ovhal_119/css/style.css]: updated

File: SubmittedAssignments/Adarsh_Ovhal_119/css/style.css (hunk 3 above)
Comment (Adarshovhal) [reply SubmittedAssignments/Adarsh_Ovhal_119/css/style.css]: updated

**Review by KalpeshApnasite — CHANGES_REQUESTED** [review]
use external css

**Pull request closed without merging**